templates/zerver/help/change-who-can-join-a-stream.md

```diff
@@ -0,0 +1,39 @@
+# Change who can join a stream
+Zulip allows you to create new private streams as a user with stream creation privileges.
+or make an existing stream private as an administrator. 
+To do so, start by navigating to the **subscriptions page**.
+
+### Finding the subscriptions page
+There are multiple ways to navigate to the subscriptions page. You can either:
+
+1. Click the **cog** (![cog](/static/images/help/cog.png)) icon in the top right corner.
+2. Select the **Manage Streams** option from the dropdown menu.
+
+![settings](/static/images/help/settings-dropdown-streams.png)
+
+or
+
+1. Click the **cog** (![cog](/static/images/help/cog.png)) icon located in the **left sidebar** next to **Streams**
+
+![Streams Cog](/static/images/help/streams-1.png)
+
+### Make an existing stream private
+To make an existing stream private you must be an administrator. Once you are on the **subscriptions page**:
+
+1. Select the stream you want to edit by either searching for its name or finding it in the streams list.
+2. Under the **Stream Settings**, find the "**Administrator settings**" section.
+3. Finally click the "**Make stream private**" button.
+
+<img src="/static/images/help/admin-make-stream-private.png" alt="Make stream private" style="width: 50%;"/>
+
+### Make a new private stream 
+Zulip allows users with stream creation privileges to create their own private streams. Once you are on the **subscriptions page**:
+
+1. Click the "<h5 style="display: inline; font-size:20px; color: #000000;">+</h5>" sign to create a new stream.
```

Comment:
Oh I see, it only looks that way on GitHub. You can see how the guide looks [here](http://1niels.zulipdev.org:9991/help/change-who-can-join-a-stream) or in the following screenshot: ![inline](https://cloud.githubusercontent.com/assets/9834624/21454119/cec3ef52-c8e3-11e6-889c-5ce6649fc282.png). Also, one problem I've noticed with adding screenshots for icons is that the drop shadow outlines the image making it look a bit out of place.